core/jazz_ui/src/app/pages/services-list/services-list.component.ts

```diff
@@ -176,6 +175,9 @@ private intervalSubscription: Subscription;
         id: service.id,
         data: service
       };
+      if(service.deployment_accounts){
+        serviceRow['platform'] = service.deployment_accounts[0].provider
+      }
```

Comment:
Do we need a default here?

Reply:
No, service should have deployment_accounts as part of the service details. Remove `if` block completely.

Reply:
No, we should remove `if` check completely. service is expected to contain this information anyways (required field during service creation).

Reply:
@raghits @devsatishm It is actually for the core services which are created during stack creation. I checked in the azure as well as sls-app stack. The core services doesn't have the deployment_accounts field. And for azure, now we have a new field in service list page called platform and since core service are missing the required field, so when in UI I open the any core services UI breaks. That's why this change.

Reply:
I think we have discussed about the `platform` field. There is no need to introduce this field & we should use `deployment_accounts` information to identify the provider information. If this requires installer changes (send `deployment_accounts` as part of bootstrapping for core services), we should make those changes in the installer.

Reply:
Done. Removed the if else block. Services are expected to have now deployment_accounts field